vmdb/app/models/miq_provision_workflow.rb

```diff
@@ -1260,7 +1260,7 @@ def ws_template_fields(values, fields, ws_values)
     unless placement_cluster_name.blank?
       data[:placement_cluster_name] = placement_cluster_name.to_s.downcase
       $log.info "#{log_header} placement_cluster_name:<#{data[:placement_cluster_name].inspect}>"
-      data[:data_centers] = EmsCluster.find(:all, :conditions => ["lower(name) = ?", data[:placement_cluster_name]]).collect {|c| c.v_parent_datacenter}
+      data[:data_centers] = EmsCluster.where("lower(name) = ?", data[:placement_cluster_name]).collect { |c| c.v_parent_datacenter }
```

Comment:
Could use `.includes(:v_parent_datacenter)`, then at the end you can do `.collect(&:v_parent_datacenter)`